MotionPhotoReader/app/src/main/java/com/google/android/libraries/motionphotoreader/BufferProcessor.java

```diff
@@ -123,7 +133,7 @@ public void process(Bundle messageData) {
         int key = messageData.getInt("MESSAGE_KEY");
         Bundle bufferData;
         int bufferIndex;
-        long timestamp;
+        long timestampUs;
```

Comment:
[commenting here since I can't comment above]

Why do we have a generic process with a Bundle argument? That is generally a pattern to be avoided, could you not have two methods nextFrame and seekTo instead?

Reply:
These are called from the nextFrame and seekTo methods, but I'll add a note to go back and potentially refactor on a separate CL.